cmake/find/FindNASM.cmake

```diff
@@ -0,0 +1,35 @@
+# This module will set `NASM_FOUND` if NASM is found.
+# If NASM is found, `NASM_ROOT` will be set to the root folder.
+# `NASM_BIN_DIR` will be set to the folder containing the executable,
+# and `NASM_EXECUTABLE` will be set to the path of the NASM executable.
+
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindNASM module")
+endif()
+
+# Try to find it even if it may already be set.
+find_path(
+  NASM_ROOT
+  NAMES
+  bin/nasm
+  bin/nasm.exe
+  )
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(
+  NASM
+  DEFAULT_MSG
+  NASM_ROOT
+  )
+
+if(NASM_FOUND)
```

Comment:
> there is no need to build hunter by oneself if there is already one

Actually that's conflict with Hunter philosophy:
* https://docs.hunter.sh/en/latest/faq/interaction-with-other-package-managers.html

> But this script is rather independent from hunter

If you want to use this module with Hunter and without you can protect Hunter checks with `if(HUNTER_ENABLED)`:
* https://docs.hunter.sh/en/latest/reference/user-variables.html#hunter-enabled